pkg/odogenerator/ingress.go

```diff
@@ -0,0 +1,119 @@
+package odogenerator
+
+import (
+	"fmt"
+	"github.com/devfile/library/pkg/devfile/generator"
+	"k8s.io/api/networking/v1"
+	"k8s.io/api/networking/v1beta1"
+)
+
+// TODO: These functions are replicated from devfile library generators and it makes more sense that they reside there
```

Comment:
should we open a PR to make then public on library end?

Reply:
Yes, we should. This is just so we we have what we need right now